packages/docusaurus-1.x/lib/server/__tests__/liveReloadServer.test.js

```diff
@@ -20,7 +20,10 @@ describe('get reload script', () => {
   test('when server started, returns url with correct port', () => {
     const port = 1234;
     liveReloadServer.start(port);
-    const expectedUrl = `http://localhost:${port}/livereload.js`;
+    const expectedUrl =
+      process.platform === 'win32'
```

Comment:
On macOS, `process.platform` is `'darwin'` and the host I'll get is 0.0.0.0, which probably wouldn't work.

Reply:
@yangshun Which do you think would be better? 

Could account for all options on [this list](https://nodejs.org/api/process.html#process_process_platform) or pass it in with a [cli flag](https://docusaurus.io/docs/en/commands#using-arguments) or [env variable](https://docusaurus.io/docs/en/commands#docusaurus-start).

Reply:
I think CLI would be better. cc @endiliey

Reply:
Talked with @endiliey. Will implement CLI flag once this is fixed: https://github.com/facebook/Docusaurus/issues/1427#issuecomment-493341612

Reply:
Make it like this
https://github.com/facebook/Docusaurus/blob/89ef4b9c440b596077249b8a69ac413187e0cac5/packages/docusaurus/bin/docusaurus.js#L82